data/mapping_Location_v4.json

```diff
@@ -1,409 +1,409 @@
+    { "id": 131,"formId":"0x0001751d","tsp":160},
+    { "id": 132,"formId":"0x00026ebd","tsp":252},
+    { "id": 133,"formId":"0x00017508","tsp":154},
+    { "id": 134,"formId":"0x0007812d","tsp":235},
+    { "id": 135,"formId":"0x000174c4","tsp":138},
+    { "id": 136,"formId":"0x000174bd","tsp":238},
+    { "id": 137,"formId":"0x00017510","tsp":150},
+    { "id": 138,"formId":"0x000174d8","tsp":239},
+    { "id": 139,"formId":"0x000174cc","tsp":145},
+    { "id": 140,"formId":"0x000174e8","tsp":126},
+    { "id": 141,"formId":"0x00017520","tsp":124},
+    { "id": 142,"formId":"0x00017504","tsp":315},
+    { "id": 143,"formId":"0x0001750c","tsp":3},
+    { "id": 144,"formId":"0x00017515","tsp":11},
+    { "id": 145,"formId":"0x000174d5","tsp":115},
+    { "id": 146,"formId":"0x0006d76c","tsp":323},
+    { "id": 147,"formId":"0x000174f0","tsp":307},
+    { "id": 148,"formId":"0x000174c8","tsp":325},
+    { "id": 149,"formId":"0x000c431a","tsp":108},
+    { "id": 150,"formId":"0x000174dd","tsp":270},
+    { "id": 151,"formId":"0x000174d9","tsp":91},
+    { "id": 152,"formId":"0x000174e4","tsp":16},
+    { "id": 153,"formId":"0x000174fa","tsp":93},
+    { "id": 154,"formId":"0x000174e9","tsp":28},
+    { "id": 155,"formId":"0x0008dc45","tsp":21},
+    { "id": 156,"formId":"0x000174e1","tsp":105},
+    { "id": 157,"formId":"0x0001fe0d","tsp":29},
+    { "id": 158,"formId":"0x0001751c","tsp":24},
+    { "id": 159,"formId":"0x000174f1","tsp":72},
+    { "id": 160,"formId":"0x000174fd","tsp":79},
+    { "id": 161,"formId":"0x000174f6","tsp":284},
+    { "id": 162,"formId":"0x000174e0","tsp":34},
+    { "id": 163,"formId":"0x000174c1","tsp":291},
+    { "id": 164,"formId":"0x000174c9","tsp":42},
+    { "id": 165,"formId":"0x00017518","tsp":287},
+    { "id": 166,"formId":"0x000174f9","tsp":68},
+    { "id": 167,"formId":"0x000174ed","tsp":57},
+    { "id": 168,"formId":"0x000174e5","tsp":46},
+    { "id": 169,"formId":"0x000174c0","tsp":61},
+    { "id": 170,"formId":"0x000174d1","tsp":52},
+    { "id": 171,"formId":"0x000cd414","tsp":206},
+    { "id": 172,"formId":"0x0000cd09","tsp":211},
+    { "id": 173,"formId":"0x00014d1a","tsp":194},
+    { "id": 174,"formId":"0x0000cef7","tsp":186},
+    { "id": 175,"formId":"0x000038cc","tsp":180},
+    { "id": 176,"formId":"0x000039ca","tsp":169},
+    { "id": 177,"formId":"0x000039d3","tsp":166},
+    { "id": 178,"formId":"0x0000cefb","tsp":232},
+    { "id": 179,"formId":"0x000038fe","tsp":159},
+    { "id": 180,"formId":"0x0000cefd","tsp":134},
+    { "id": 181,"formId":"0x00003967","tsp":120},
+    { "id": 182,"formId":"0x00003937","tsp":114},
+    { "id": 183,"formId":"0x0000cef9","tsp":111},
+    { "id": 184,"formId":"0x00003986","tsp":272},
+    { "id": 185,"formId":"0x0000cefc","tsp":76},
+    { "id": 186,"formId":"0x0000cef8","tsp":297},
+    { "id": 187,"formId":"0x000038a9","tsp":78},
+    { "id": 188,"formId":"0x000038b4","tsp":70},
+    { "id": 189,"formId":"0x00014d15","tsp":-1},
```

Comment:
-1 tsp is a placeholder, could probably be replaced with a undefined check, but it made sorting and removing the extra TSP points easy down the line.

Reply:
Yeah. We use undefined checks in the other files, so I think it would be good to use them here.

Reply:
will create an undefined check in map.js then.
and remove -1 tspIDs